app/views/posts/show.html.erb

```diff
@@ -26,5 +31,15 @@
         <h3>Like this post</h3>
         <%= render 'likes/form', like: @like %>
       </div>
+      <div class="button_holder">
+      <% if can? :destroy, @post%>
+        <button>
+          <%= link_to 'Delete post', user_post_path(@current_user,@post), data: {
+                      turbo_method: :delete,
+                      turbo_confirm: "Are you sure?"
+                    } %>
+        </button>
```

Comment:
Deleting a post also throws an error. Kindly make sure that the owner can delete their post successfully.

- However, the post does get removed from the page but the post count is not updated. Please make sure there's no errors upon deletion and the post counter is updated.

![image](https://github.com/AyoMoses1/Gems_Blog/assets/38088818/d061f591-7aec-47ae-a203-f27debeb8ef9)

Reply:
This change has not yet been done and the error persists